cmd/cli/zip.go

```diff
@@ -12,10 +22,58 @@ type file struct {
 }
 
 // the dir must end with a "/"
-func makeZip(dir string) ([]byte, error) {
+func makeZip(dir, basePath, module, version string) ([]byte, error) {
+	ignoreParser := getIgnoreParser(dir)
 	buf := new(bytes.Buffer)
-	if err := zip.Archive(dir, buf, nil); err != nil {
-		return nil, err
+	w := zip.NewWriter(buf)
+
+	walkFunc := func(path string, info os.FileInfo, err error) error {
```

Comment:
I filed https://github.com/gomods/athens/issues/31 to remind us to test this later